packages/harpguru-core/src/types/index.ts

```diff
@@ -1,5 +1,15 @@
 import type { HarpStrata } from 'harpstrata'
-import type { ApparatusIds, PitchIds, PozitionIds } from 'harpparts'
+import type { ApparatusIds, DegreeIds, PitchIds, PozitionIds } from 'harpparts'
+import type { CovariantMembers } from 'harpcovariance'
+
+export type GlobalState = {
```

Comment:
We can move this back to the HarpGuru component utils now that it's not needed for import elsewhere.

Reply:
No longer true. The dispatchers need it.